example/spec/syncGranule/SyncGranule.output.payload.template.json

```diff
@@ -10,15 +10,15 @@
           "filename": "",
           "fileSize": 17865615,
           "path": "cumulus-test-data/pdrs",
-          "url_path": "{cmrMetadata.Granule.GranuleUR}/{file.name}"
+          "url_path": "{cmrMetadata.Granule.Collection.ShortName}/{extractYear(cmrMetadata.Granule.Temporal.RangeDateTime.BeginningDateTime)}/{substring(file.name, 0, 3)}"
```

Comment:
We know what cmrMetadata is, right? Shouldn't we be able to hardcode these for testing the output?